### PR TITLE
fix: colony details styles, new native token pill

### LINKED ISSUE #1389: Colony details page: description + slug fixes
## Description

A couple of fixes are required to the Colony Details page that are outstanding from [PR #1348 ](https://github.com/JoinColony/colonyCDapp/pull/1348). These include:

1. The full Colony description should show without the ellipses. There is a max character limit inherited from the edit Colony Details action type. 

image

2. The full Colony name should show in the title and not the slug. 

3. The native token widget should be the same height and padding as the desktop version:
image

4. We should match the design of both token widgets 

**Colony details page:**
image

**Dashboard:**
image

### PR DESCRIPTION
**Pull request by bassgeta**
## Description

Fix styles according to the issue.
Added a new native token pill so we can reuse it between the details page and the dashboard header.

**New stuff** ✨

* `NativeTokenPill` component which displays a tooltip 

**Changes** 🏗

* Use the new component instead of doing it twice

Resolves  #1389

### CODE REVIEW

File: src/components/v5/common/NativeTokenPill/NativeTokenPill.tsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+}
+
+const displayName = 'v5.shared.NativeTokenPill';
+
+const NativeTokenPill = ({
+  tokenName,
+  isLocked = false,
+}: NativeTokenPillProps) => {
+  return (
+    <div className="h-[1.875rem] flex flex-row items-center px-[.375rem] border border-gray-200 rounded-lg bg-base-white text-gray-900">
```
Comment (chmanie) [inline src/components/v5/common/NativeTokenPill/NativeTokenPill.tsx:19]: 0.375rem is `px-1.5`

File: src/components/v5/common/NativeTokenPill/NativeTokenPill.tsx [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+      <span className="text-sm font-medium">{tokenName}</span>
+      {isLocked && (
+        <Tooltip
+          tooltipContent={
+            <span>{formatText({ id: 'tooltip.lockedToken' })}</span>
+          }
+        >
+          <LockKey size={12} className="ml-[.125rem] -translate-x-px" />
```
Comment (chmanie) [inline src/components/v5/common/NativeTokenPill/NativeTokenPill.tsx:27]: 0.125rem is `ml-0.5`

**Review by melyndav — CHANGES_REQUESTED** [review]
Hey @bassgeta,

You've changed the desktop styling of the native token pill. This shouldn't match, they have different styling. 

Link to dashboard: https://www.figma.com/file/5V8pr7iMwXsT9L3VAZsmUt/Colony-V5?type=design&node-id=11846-72482&mode=design&t=j9QKipjvSqXI1GXM-4

Also, double check in figma the padding between the text and the icon inside the pill. It's looking a little small.

**Review by rdig — APPROVED** [review]
code wise I'm good with this. As long as Mel's UI updates are covered, this is good to go

**Review by melyndav — APPROVED** [review]
You know the drill, get this one shipppppped! Nice work @bassgeta, I appreciate you working on the UI fixes as requested.

**Pull request merged**